src/System.Runtime.Serialization.Xml/pkg/System.Runtime.Serialization.Xml.pkgproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <ProjectReference Include="..\ref\System.Runtime.Serialization.Xml.csproj">
-      <SupportedFramework>net46;netcore50;netcoreapp1.0;$(AllXamarinFrameworks)</SupportedFramework>
+      <SupportedFramework>net463;netcoreapp1.1;$(AllXamarinFrameworks)</SupportedFramework>
```

Comment:
@weshaggard I'm not sure about the answer. It worked on netcore50. Does that mean it should work on uap10.1?

---
In reply to: [82297034](https://github.com/dotnet/corefx/pull/12378#discussion_r82297034) [](ancestors = 82297034)